src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SnsService.kt

```diff
@@ -102,7 +103,7 @@ class SnsService(
         null,
       )
     } catch (e: Throwable) {
-      throw PublishEventException("Failed to publish Event $payloadEvent.eventType to $TOPIC_ID", e)
+      throw PublishEventException("Failed (publishToDomainEventsTopic) to publish Event $payloadEvent.eventType to $TOPIC_ID", e)
```

Comment:
does this need to be ${payloadEvent.eventType}?

Reply:
It's just to group the log above to this one,... so on insights I can just search for publishToDomainEventsTopic